### ISSUE TITLE
Some Duotone Icons do not make sense

### ISSUE THREAD

**Issue by diesl** [FortAwesome/Font-Awesome#15833]
**Describe the bug**
Although most Duotone Icons are great, some Duotone Icons do not make sense or are even misleading in some cases. 

A good example is the [Mathematic section](https://fontawesome.com/icons?c=mathematics). In my opinion, it makes no sense to split greek letters or (mathematic) symbols into parts with different colors. Would you do it with letters from latin alphabet?

- [Equals](https://fontawesome.com/icons/equals?style=duotone): One of the best examples. Looks like a underlined Minus?
- [Plus](https://fontawesome.com/icons/plus?style=duotone): Looks like Minus, striked out. Even next to a proper Minus Icon, it looks like Minus with different opacity. At least, it is misleading to the eye
- [Times](https://fontawesome.com/icons/times?style=duotone): Looks like Divide "/", striked out. Or Difference "\" (set theory), striked out?

Another non-mathematic example:
- [Check](https://fontawesome.com/icons/check?style=duotone): I think you get it. It makes simply no sense to have two colors here

Greek letter example:
- [Lambda](https://fontawesome.com/icons/lambda?style=duotone): It is a letter, should not be bicolored.

Other mathematic icons like [divide](https://fontawesome.com/icons/divide?style=duotone) or [percentage](https://fontawesome.com/icons/percentage?style=duotone) are a bit less misleading, but only because the are not that easily confounded with other symbols.

If you argue that Duotone must be duotoned, then let's look at these Icons:

- [Wave-square](https://fontawesome.com/icons/wave-square?style=duotone): No Duotone
- [Square root](https://fontawesome.com/icons/square-root?style=duotone): No Duotone

**To Reproduce**
n/a

**Expected behavior**
The mentioned icons should be colored with only one color, or have a totally different approach, for example like [Pause](https://fontawesome.com/icons/pause?style=duotone). A different argument is inconsistency with other mathematic symbols that are not bicolored.

**Screenshots**
See linked icons.

**Version and implementation**
Version: 5.11.2
Browser and version: n/a

- [x] SVG with JS
- [x] Web Fonts with CSS
- [x] SVG Sprites
- [x] On the Desktop

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [x] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for this detailed report.

I'm going to assign this to @sensibleworld, but do you have in mind a nicer title for this request?

**Comment by diesl** [comment]
> do you have in mind a nicer title for this request?

Yes, after writing down the complete issue, the title seems a bit unspecific. What do you think of "Letters and symbols should not be duotoned"? Feel free to change it to a more appropriate title.

**Comment by tagliala** [comment]
> "Letters and symbols should not be duotoned"

Yes, I'm going to change to something like that

**Comment by mr21** [comment]
We just have to set the `opacity: 1` to the icon's `::after` we desire...
It would be weird to remove it for everyone...

**Comment by diesl** [comment]
> We just have to set the `opacity: 1` to the icon's `::after` we desire...
> It would be weird to remove it for everyone...

Having only one color instead of two is something different from setting opacity to 1.

I suggest to use only 1 color in cases where 2 colors do not make sense. There are already icons that have only one color in Duotone, e.g. wave-square and others (see issue description).

**Comment by mr21** [comment]
Yes, but you can still do 

```
.fad-plop::after {
 opacity: 1;
}
```

After this, your icons will look like a solid one.

**Comment by diesl** [comment]
> After this, your icons will look like a solid one.

Ah sorry, now I got it. Yes, you are right, but still I think it should be consistent across all icons. And to be honest, I also am not keen on having to write this extra CSS.

**Comment by diesl** [comment]
Hi, are there any news about this ticket?

**Comment by sensibleworld** [comment]
@diesl Yup! There are a bunch of changes coming to the duotone icons in version 6, and you'll see more that follow your suggestions above. We're releasing an alpha soon, so folks can start testing things out. I fully expect we'll need to make more duotone changes before the official 6 release (I guarantee I missed some updates!), so feedback during the alpha stages is going to be key.

**Comment by diesl** [comment]
Font Awesome 6 has been released some time ago, but unfortunately this ticket did not make it. 

@sensibleworld Therefore I wanted to ask again, how and when will this be fixed? Is it somewhere on the roadmap?
